pyTigerGraph/pyTigerGraphBase.py

```diff
@@ -128,10 +127,11 @@ def __init__(self, host: str = "http://127.0.0.1", graphname: str = "MyGraph",
 
         self.tgCloud = tgCloud or gcp
         if "tgcloud" in self.netloc.lower():
-            try: # if get request succeeds, using TG Cloud instance provisioned before 6/20/2022
-                self._get(self.host + ":" + str(restppPort) + "/echo/" + graphname, resKey="message")
-            except requests.exceptions.RequestException: # if get request fails, using TG Cloud instance provisioned after 6/20/2022, after new firewall config
+            try: # if get request succeeds, using TG Cloud instance provisioned after 6/20/2022
+                self._get(self.host + "/api/ping", resKey="message")
```

Comment:
Does it fail fast if the instance is not on new tgcloud?

Reply:
Yes it does, confirmed on a box from pre-6/20/2022